templates/default/abiquo.conf.erb

```diff
@@ -29,21 +33,30 @@
         ProxyPassReverse ajp://localhost:<%= node['abiquo']['tomcat']['ajp-port'] %>/api
     </Location>
 
+<% if node['abiquo']['profile'] == 'monolithic' %>
     <Location /am>
         ProxyPass ajp://localhost:<%= node['abiquo']['tomcat']['ajp-port'] %>/am
         ProxyPassReverse ajp://localhost:<%= node['abiquo']['tomcat']['ajp-port'] %>/am
     </Location>
- 
+ <% end %>
```

Comment:
We could consider appending the proxy to the proxies list in the monolithic setup, to avoid having this here.